events/admin.py

```diff
@@ -160,6 +163,12 @@ def save_model(self, request, obj, form, change):
         obj.events.set(form.cleaned_data["events"])
 
     def publish(self, request, queryset):
+        success_count = 0
         for obj in queryset:
-            obj.publish()
-        self.message_user(request, _("%s successfully published.") % queryset.count())
+            try:
+                obj.publish()
```

Comment:
So, if multiple event groups are published, but one of them raises an exception, I guess it still okay to publish the ones which do not cause an exception?

Reply:
Since publishing will send notifications, this logic here is fine. 👍

Reply:
Yup good point. Ideally if one fails the whole thing should fail, but event group publishing will send notifications, so that would not be really nice. Also in practice, it is highly unlikely multiple event groups will be ever published at the same time anyway, so this should be fine 🙂